examples/gnrc_networking/README.md

```diff
@@ -58,12 +62,12 @@ Now, on the Linux host, you can run netcat to connect with RIOT's UDP server:
 
     nc -6uv fe80::ccf5:e1ff:fec5:f75a%tap0 8808
 
-The `-6` option is necessary to tell netcat to use IPv6 only, the `-u` option tells
-it to use UDP only, and the `-v` option makes it give more verbose output (this one is optional).
+The `-6` option is necessary to tell netcat to use IPv6 only, the `-u`
+option tells it to use UDP only, and the `-v` option makes it give more verbose output (this one is optional).
```

Comment:
Is this line not broken intentionally?

Reply:
If so, I'm afraid I don't understand the intent :/

Reply:
Then please break it. It is definitely >72 chars (or is github weird again?)